src/st/no3/48A84.c

```diff
@@ -1,506 +1,561 @@
-            newEnt->drawFlags = FLAG_DRAW_ROTY | FLAG_DRAW_ROTX;
-            newEnt->zPriority = entity->zPriority + 1;
-        }
-    }
-}
-
-void func_801CF58C(Entity* self) {
-    self->velocityX = 0;
-    self->ext.generic.unk84.S16.unk2 = 0x100;
-    SetStep(6);
-    g_api.PlaySfx(SE_WARG_GROWL);
-    self->ext.generic.unk80.modeS16.unk0 = 0x20;
-}
-
-void func_801CF5E0(Entity* self) {
-    s16 temp_v0;
-
-    if (self->facingLeft == GetSideToPlayer()) {
-        SetStep(5);
-        return;
-    }
-
-    if (self->ext.generic.unk84.S16.unk2 == 0) {
-        func_801CF58C(self);
-        return;
-    }
-
-    temp_v0 = self->ext.generic.unk84.S16.unk0 - self->posX.i.hi -
-              g_Tilemap.scrollX.i.hi;
-
-    if (temp_v0 > 16) {
-        SetStep(3);
-        if (self->facingLeft != 0) {
-            self->ext.generic.unk7C.S8.unk0 = 0;
-        } else {
-            self->ext.generic.unk7C.S8.unk0 = 1;
-        }
-    } else if (temp_v0 < -16) {
-        SetStep(3);
-        if (self->facingLeft != 0) {
-            self->ext.generic.unk7C.S8.unk0 = 1;
-        } else {
-            self->ext.generic.unk7C.S8.unk0 = 0;
-        }
-    } else {
-        SetStep(7);
-    }
-
-    self->ext.generic.unk80.modeS16.unk0 = 0;
-    self->ext.generic.unk80.modeS16.unk2 = 32;
-}
-
-// duplicate of func_801CC90C in this file
-void func_801CF6D8(Entity* arg0) {
-    s16 temp_v0_2;
-    s16 temp_v1;
-    s16 temp_v0;
-
-    temp_v0 = GetDistanceToPlayerX();
-    temp_v1 = arg0->ext.generic.unk84.S16.unk2;
-    if (temp_v1 != 0) {
-
-        if ((u32)(temp_v0) < 0x60) {
-            temp_v0_2 = temp_v1 - 2;
-            arg0->ext.generic.unk84.S16.unk2 = temp_v0_2;
-            if (temp_v0_2 < 0) {
-                arg0->ext.generic.unk84.S16.unk2 = 0;
-            }
-        } else {
-            arg0->ext.generic.unk84.S16.unk2 = (temp_v1 - 1);
-        }
-    }
-}
-
-INCLUDE_ASM("st/no3/nonmatchings/48A84", EntityWarg);
-
-INCLUDE_ASM("st/no3/nonmatchings/48A84", EntityUnkId4B);
-
-// A single "puff" of the warg explosion animation, transparent
-void EntityWargExplosionPuffTransparent(Entity* entity) {
-    u32 temp_v0;
-
-    if (entity->step == 0) {
-        InitializeEntity(g_InitializeEntityData0);
-        entity->animSet = ANIMSET_DRA(14);
-        entity->unk5A = 0x79;
-        entity->palette = 0xD0;
-        entity->drawMode = DRAW_TPAGE2 | DRAW_TPAGE;
-        entity->drawFlags = FLAG_DRAW_UNK8;
-        entity->animFrameIdx = 0;
-        entity->animFrameDuration = 0;
-        entity->unk6C = 0x60;
-        temp_v0 = entity->params & 0xFF00;
-        if (temp_v0 != 0) {
-            entity->zPriority = temp_v0 >> 8;
-        }
-        entity->velocityY += 0xFFFF0000;
-        return;
-    }
-    entity->posY.val += entity->velocityY;
-    if (AnimateEntity(D_8018383C, entity) == 0) {
-        DestroyEntity(entity);
-    }
-}
-
-#include "../popup.h"
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "no3.h"

```

Comment:
Something is wrong with your whitespace and it's messing up the diff, do you have windows newlines or something?

Reply:
Must be that yeah, I thought `make format` would have fixed that but I guess not, I'll update.